### ISSUE TITLE
TUI ignore terminal resizing

### ISSUE THREAD

**Issue by RomanKharin** [pfalcon/ScratchABit#13]
TUI ignore terminal resizing

Occured on gnome-terminal, xterm

btw Is there "redraw screen" hotkey?

**Comment by pfalcon** [comment]
That's true. Terminal (XTerm) is sized once at startup and this size is used afterwards. May be possible to fix it eventually, but low enough priority.

The whole screen us redrawn for example when opening and then closing menu popup (this won't lead to new terminal size being picked up, as explained above).
